app/_hub/impart-security/kong-plugin-impart/_metadata/_index.yml

```diff
@@ -0,0 +1,41 @@
+name: Kong Plugin Impart
```

Comment:
Suggestion: change the name to `Impart Security` 

I would recommend removing "kong plugin" from the name. This is the field that will be used to create the title on the card.

Reply:
Thanks!